src/com/shapestone/travelportal/Payment.java

```diff
@@ -0,0 +1,36 @@
+package com.shapestone.travelportal;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class Payment {
+	public static void main(String... args) throws StreamReadException, DatabindException, IOException {
+		ObjectMapper om = new ObjectMapper();
+		Booking[] booking = om.readValue(new java.io.File("travel.json"), Booking[].class);
+		TotalAmount[] amount = om.readValue(new java.io.File("travelportal.json"), TotalAmount[].class);
+		for (int j = 0; j <= booking.length - 1; j++) {
```

Comment:
Print results in Table format

Reply:
Changes are done. Please review the Changes